doc/sphinx-guides/source/container/base-image.rst

```diff
@@ -281,6 +281,18 @@ provides. These are mostly based on environment variables (very common with cont
       - See :ref:`:ApplicationServerSettings` ``http.request-timeout-seconds``.
 
         *Note:* can also be set using any other `MicroProfile Config Sources`_ available via ``dataverse.http.timeout``.
+    * - ``DATAVERSE_COOKIE_SAME_SITE_VALUE``
```

Comment:
When I try to change DATAVERSE_COOKIE_SAME_SITE_VALUE to "Strict" in the compose file, it doesn't seem to have an effect. I still see "Lax" when I check the cookie with curl.

I opened a thread in Zulip about this: https://dataverse.zulipchat.com/#narrow/channel/375812-containers/topic/DATAVERSE_HTTP_TIMEOUT.20not.20having.20an.20effect/near/499094581

I'm not sure why it isn't working. 🤷 I'm hoping @poikilotherm will save me! ❤️

Reply:
Did you also set DATAVERSE_COOKIE_SAME_SITE_ENABLED?

Reply:
Well, it's already set.

Reply:
Through the Dockerfile, I mean.

Reply:
Hold on, I need to add something to the Dockerfile.

Reply:
Bah. It doesn't work. As discussed at standup, I switched Docker to be hard coded to Lax and updated the docs to say you have to rebuild the base image if you want Strict or whatever. See d62ac2f8df